test/e2e/apimachinery/flowcontrol.go

```diff
@@ -51,21 +51,26 @@ var _ = SIGDescribe("API priority and fairness", func() {
 		matchingUsername := "noxu"
 		nonMatchingUsername := "foo"
 
-		ginkgo.By("creating a testing prioritylevel")
+		ginkgo.By("creating a testing priority-level")
 		createdPriorityLevel, cleanup := createPriorityLevel(f, testingPriorityLevelName, 1)
 		defer cleanup()
 
-		ginkgo.By("creating a testing flowschema")
+		ginkgo.By("creating a testing flow-schema")
 		createdFlowSchema, cleanup := createFlowSchema(f, testingFlowSchemaName, 1000, testingPriorityLevelName, matchingUsername)
 		defer cleanup()
 
-		ginkgo.By("checking response headers contain flow-schema/priority-level uid")
-		if !testResponseHeaderMatches(f, matchingUsername, string(createdPriorityLevel.UID), string(createdFlowSchema.UID)) {
-			framework.Failf("matching user doesnt received UID for the testing priority-level and flow-schema")
-		}
-		if testResponseHeaderMatches(f, nonMatchingUsername, string(createdPriorityLevel.UID), string(createdPriorityLevel.UID)) {
```

Comment:
@MikeSpreitzer it seems we have been passing `createdPriorityLevel.UID` for both flow-schema and priority-level. If I am not mistaken, for a non matching user, we expect the UID of catch-all objects in the response header.

Below, I am validating that a non matching user gets a catch-all UID in the response header. Let me know if this looks good.